### PR TITLE
Nerfs telekinesis 

### PR DESCRIPTION
**Pull request by GDNgit**
## What Does This PR Do
Nerfs the cooldown of the TK genetic ability from .3 seconds to 3 seconds

## Why It's Good For The Game
TK already does a lot; it lets you use items from range such as a flash and allows you to interact with some objects from a distance. However, TK is mostly used for its power in combat as you can very quickly destroy structures/kill mobs due to its extremely low cooldown. Its cooldown is being raised to 3 seconds from .3 seconds so that it still has the ability to move items for purpose of utility, but it's less powerful for combat related shenanigans. 

## Images of changes
Video format not supported; I'll upload an MP4 in a bit 

## Changelog
:cl:
balance: TK nerfed from .3 seconds to 3 second use time delay 
/:cl:

### CODE REVIEW

**Comment by GDNgit** [comment]
Annddd this is why you don't let github do it's thing, ty Github

File: code/_onclick/telekinesis.dm [hunk 1]
```diff
@@ -104,7 +104,7 @@
 /obj/item/tk_grab/afterattack(atom/target , mob/living/user, proximity, params)//TODO: go over this
 	if(!target || !user)
 		return
-	if(last_throw+3 > world.time)
+	if(last_throw+30 > world.time)
```
Comment (Sirryan2002) [inline code/_onclick/telekinesis.dm:107]:
```suggestion
	if(last_throw + 3 SECONDS > world.time)
```

**Review by Sirryan2002 — COMMENTED** [review]
you may want to fix your branch :eyes:

File: code/_onclick/telekinesis.dm (hunk 1 above)
Comment (GDNgit) [reply code/_onclick/telekinesis.dm:107]: Yep, it's even the OLD version, I should probably just rePR this

**Comment by GDNgit** [comment]
Closing so I can fix my damn branch in the least painful way possible

**Pull request closed without merging**